msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -1521,6 +1521,131 @@ The following code provides an example of meeting end event payload:
 | **value.EndTime** | The meeting end time in UTC. |
```

Comment:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/95664393/29f6af83-e4fa-4291-a1aa-35ca5e7e595d)
This section could be updated from " Get real-time Teams meeting events API" to "Receive meeting start/end events" for a more correct description. 

Also in the note should use "meeting start/end events"  instead of  "real time meeting events"

Reply:
Updates